ale/base/data_naif.py

```diff
@@ -363,18 +363,22 @@ def ephemeris_start_time(self):
         return spice.scs2e(self.spacecraft_id, self.spacecraft_clock_start_count)
 
     @property
-    def center_ephemeris_time(self):
+    def ephemeris_stop_time(self):
         """
-        Returns the average of the start and stop ephemeris times. Expects
-        ephemeris start and stop times to be defined. These should be double precision
-        numbers containing the ephemeris start and stop times of the image.
+        Returns the ephemeris stop time of the image. Expects spacecraft_id to
+        be defined. This must be the integer Naif Id code for the spacecraft.
+        Expects spacecraft_clock_stop_count to be defined. This must be a string
+        containing the stop clock count of the spacecraft
 
         Returns
         -------
         : double
-          Center ephemeris time for an image
+          Ephemeris stop time of the image
         """
-        return (self.ephemeris_start_time + self.ephemeris_stop_time) / 2
+        if self.spacecraft_clock_stop_count:
+            return spice.scs2e(self.spacecraft_id, self.spacecraft_clock_stop_count)
+        else:
+            return spice.str2et(self.utc_stop_time.strftime("%Y-%m-%d %H:%M:%S"))
```

Comment:
This should be removed, this will cause timing errors and result in center times that don't match up with ISIS